pages/common/chisel.md

```diff
@@ -0,0 +1,28 @@
+# chisel
+
+> Chisel is a tool creating a TCP tunnel. Including both client and server.
+> More information: <https://github.com/jpillora/chisel>.
+
+- Run a chisel server:
+
+`chisel server`
+
+- Run a chisel server listening to a specific port:
+
+`chisel server -p {{server_port}}`
+
+- Run a chisel server securing the connection using a username and password authentication:
+
+`chisel server --auth {{username}}:{{password}}`
+
+- Connect to a chisel server and tunnel a specific port to a remote server and port:
+
+`chisel client {{server_ip}}:{{server_port}} {{local_port}}:{{remote_server}}:{{remote_port}}`
+
+- Connect to a chisel server and tunnel a specific host and port to a remote server and port:
+
+`chisel client {{server_ip}}:{{server_port}} {{local_host}}:{{local_port}}:{{remote_server}}:{{remote_port}}`
+
+- Connect to a chisel server using a username and password authentication:
```

Comment:
I'm not a native speaker, but I think you can see "authentication" as a singular noun and "username and password" as the plural describing the authentication. Does it make sense?

Reply:
No it's not an issue of singular/plural. The article `a` is just not needed to qualify "username and password" at all. I'm not a native speaker also. Nevertheless, I will ping our grammar experts @waldyrious/ @sbrl in case I am mistaken.

Reply:
Anyway, even if it does, it should be an `an` :thinking:

Reply:
As a native speaker, allow me to weigh in. The syntax `using username and password authentication` is indeed correct, as the `username and password` is plural - even if it's attached to the `authentication`.

@agnivade: I'm a grammar expert? Happy to be of service, but lol :P

Reply:
Thanks @sbrl. @ohadeytan - would you like to open a PR to rectify this ? If not, I can do it too. Let me know.

Reply:
[Done](https://github.com/tldr-pages/tldr/pull/3410).